add Traditional Chinese translate

tasi788 opened the pull request:
add Traditional Chinese language options

tasi788 committed:
add Traditional Chinese translate

tonghoai commented:
Hello @tasi788, thank you for supporting the language translation. But before proceeding, there are some issues as follows:

1. You need to add a translation for the display language in locales/vi.json, locales/en.json, and locales/zhtw.json with the attribute app.setting_general_language_zhtw (I might have been a bit careless with duplicate codes, but for now, this is the quickest fix).

2. I see you have translated the landing page, but the new language option is not yet displayed. Check at pages/landing/index.vue - line 156.

3. You need to display the new language option in the settings screen for mobile devices. Check at components/navbar/top/index.vue - line 334.

tasi788 committed:
Add Traditional Chinese language support in settings and landing page.

- Include "Traditional Chinese" option in language settings for English, Vietnamese, and Traditional Chinese.
- Update translations for "Traditional Chinese" in English, Vietnamese, and Traditional Chinese locales.
- Add "Traditional Chinese" language option on the landing page.

tasi788 commented:
![image](https://github.com/tonghoai/opennotas/assets/11913223/a7277ff0-888f-45e6-b286-9b4939fe3ed0)
![image](https://github.com/tonghoai/opennotas/assets/11913223/939a2dde-5730-44fd-a216-167f7007ec78)

@tonghoai now looks great, thanks a lot

tonghoai commented:
Congratulations on your first job with JavaScript!
